src/Files.App/Actions/Navigation/NavigateForwardAction.cs

```diff
@@ -19,8 +19,8 @@ public HotKey HotKey
 		public HotKey SecondHotKey
 			=> new(Keys.Mouse5);
 
-		public HotKey MediaHotKey
-			=> new(Keys.GoForward, false);
+		//public HotKey MediaHotKey
+		//	=> new(Keys.GoForward, false);
```

Comment:
Why is this commented out?

Reply:
Because I had decided not to support hidden keys.
But I will revert them

Reply:
As above.

Reply:
Uncommented out